integration-libs/cdc/schematics/add-cdc/index_spec.ts

```diff
@@ -122,6 +122,44 @@ describe('Spartacus CDC schematics: ng-add', () => {
   });
 
   describe('CDC feature', () => {
+    describe('warning for missing jsSDKUrl', () => {
+      let firstMessage: string | undefined;
+      beforeEach(async () => {
+        schematicRunner.logger.subscribe((log) => {
+          if (!firstMessage) {
+            firstMessage = log.message;
+          }
+        });
+
+        appTree = await schematicRunner
+          .runSchematicAsync('ng-add', { ...cdcFeatureOptions }, appTree)
+          .toPromise();
+      });
+
+      it('show the warning', () => {
+        expect(firstMessage).toEqual(
+          `CDC JS SDK URL is not provided. Please run the schematic again, or make sure you update the javascriptUrl.`
+        );
+      });
+    });
+
+    describe('validation of jsSDKUrl', () => {
+      beforeEach(async () => {
+        appTree = await schematicRunner
+          .runSchematicAsync(
+            'ng-add',
+            { ...cdcFeatureOptions, javascriptUrl: '<dc>.gigya.com/<api-key>' },
+            appTree
+          )
+          .toPromise();
+      });
+
+      it('should set the given javascriptUrl', () => {
+        const module = appTree.readContent(featureModulePath);
+        expect(module).toContain<string>(`<dc>.gigya.com/<api-key>`);
```

Comment:
instead of `expect(module).toContain<string>(`<dc>.gigya.com/<api-key>`);`

put `expect(module).toMatchSnapshot();`

P.S. go to `integration-libs/cdc` and run `yarn test:schematics` 
this will basically create an entry under the snapshot for this test case. This snap will help with future changes as it will detect the output of the snap is different to what it was before.

Reply:
^ snap is part of jest features

Reply:
yes, will update the test case Brian.